docs/commerce/cart/cart.md

```diff
@@ -94,7 +101,69 @@ and taxes included) in the selected currency.
 You could extend the Cart summary widget to let buyers navigate from this view, 
 for example, to checkout, or back to shopping, by adding respective buttons.
 
+### Cart events
+
+When cart data is changed or loaded, the `ibexa-cart:cart-data-changed` event is triggered on `body`.
+The reference to the Cart is sent in the event's `detail`.
+
+```js
+document.body.addEventListener(
+    'ibexa-cart:cart-data-changed',
+    ({ detail: { cart } }) => {
+        refreshMyWidget(cart);
+    },
+    false,
+);
+```
+
 ### Cart service 
 
 The Cart package provides `Ibexa\Contracts\Cart\CartServiceInterface` Symfony service, 
 which is the entrypoint for calling the [backend API](cart_api.md).
+
+You can import services using the following code:
+
+```js
+import * as cartService from '@ibexa-cart/src/bundle/Resources/public/js/service/cart';
+```
+
+Next, use in the code as follows
+
+```js
+cartService.deleteCartEntry(cartIdentifier, entryIdentifier);
+```
+
+Every cart service function returns a Promise with a parsed response.
```

Comment:
Good point, they actually use lowercase _promise_ also. I like their style of using both. They mention `Promise` to let users know that they are talking about a specific promise, but then they use _promise_ so that the text is not obfuscated with too many `Promise`s.
Another nice thing about using both forms is that the second word, _promise_, can be easily 
changed by cases, like in the previous sentence, "promises" would be easier to read, but like in the previous sentence, sometimes we need to use `Promise` to be more specific. 😄

Reply:
I think the good idea is to apply `Promise` when talking about this specific concept.